projects/default/protos/SumoInterface.proto

```diff
@@ -32,15 +32,15 @@ PROTO SumoInterface [
 {
   %{
     -- tracks check
-    local arguments = ""
+    local arguments = "[ "
     if fields.gui.value ~= true then
-      arguments = arguments .. "--no-gui "
+      arguments = arguments .. "\"--no-gui\" "
     end
     if fields.useNetconvert.value ~= true then
-      arguments = arguments .. "--no-netconvert "
+      arguments = arguments .. "\"--no-netconvert\" "
```

Comment:
Instead of a string we can use an array.
Something like (untested):
```
local arguments = {}
if fields.gui.value ~= true then
  table.insert(arguments, "--no-gui")
end
...
```

And then 'insert' the arguments with something like:
```
controllerArgs [
  %{ for index, value in ipairs(arguments) do }%
  %{= value .. '\n' }%
  %{ end }%
]
```